src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/payment/processor/config/SwaggerPublisher.java

```diff
@@ -22,6 +22,7 @@
 @SpringJUnitWebConfig
 @SpringBootTest
```

Comment:
Introduction to `@IntegrationTest` makes above ones redundant

Reply:
removed